tensorflow/core/util/onednn_env_vars.cc

```diff
@@ -53,5 +53,15 @@ bool ThreadPoolUseCallerThread() {
   return threadpool_use_caller_thread;
 }
 
+string FPMathModeSetting() {
```

Comment:
Please use std::string in this file as well.

Reply:
Updated, thank you.